pkg/interop/server_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package interop
```

Comment:
I'm never a big fan of unnecessary changes

Reply:
This was necessary during development due to import cycle.
This test is inconsistent with our guidelines, so it has to either be renamed or this change has to be made. Since I already made the change during development I don't see a reason to revert it just to perform the same change later in the future.
I can open a separate PR for this if you want :man_shrugging: